examples/convert_little_endian_to_big.cpp

```diff
@@ -30,10 +30,10 @@ int main()
     std::cout << "in is: " << std::hex << in << std::endl;
     // If the host is little endian, the put function
     // should change the byte order (no change for big endian)
-    endian::convert_endian::put32(in, data);
+    endian::little_endian::put32(in, data);
```

Comment:
I think the name of the example should be `convert_to_little_endian` (if that was your intention by calling `endian::little_endian::put32`)